features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/db2.sql

```diff
@@ -1896,6 +1896,7 @@ CREATE TABLE AM_EXTERNAL_STORES (
             STORE_DISPLAY_NAME VARCHAR(255) NOT NULL,
```

Comment:
Seems we have introduced some modifications to the schema. So this has a migration impact right? So we have to track this.

Reply:
Yes this has an impact. Created https://github.com/wso2/product-apim/issues/5505 to track the schema migration changes.

Reply:
:+1: